core/src/main/java/com/klaviyo/coresdk/Klaviyo.kt

```diff
@@ -6,13 +6,13 @@ import com.klaviyo.coresdk.networking.requests.KlaviyoRequest
 import com.klaviyo.coresdk.networking.requests.TrackRequest
 
 object Klaviyo {
-    fun track(event: KlaviyoEvent, customerProperties: Map<String, String>, properties: Map<String, String>? = null) {
+    fun track(event: KlaviyoEvent, customerProperties: MutableMap<String, String>, properties: Map<String, String>? = null) {
```

Comment:
That feels weird assuming this is a public api. It's ok if you have to copy the map to new one to modify it.

Reply:
+1